kie-server-parent/kie-server-tests/kie-server-integ-tests-drools/src/test/java/org/kie/server/integrationtests/drools/pmml/ApplyScorecardMultiModuleIntegrationTest.java

```diff
@@ -0,0 +1,82 @@
+package org.kie.server.integrationtests.drools.pmml;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.drools.core.command.impl.CommandFactoryServiceImpl;
+import org.drools.core.command.runtime.pmml.ApplyPmmlModelCommand;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.kie.api.pmml.PMML4Result;
+import org.kie.api.pmml.PMMLRequestData;
+import org.kie.api.runtime.ExecutionResults;
+import org.kie.scanner.KieMavenRepository;
+import org.kie.scanner.KieURLClassLoader;
+import org.kie.server.api.model.KieContainerResource;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.api.model.ServiceResponse;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.integrationtests.shared.KieServerAssert;
+import org.kie.server.integrationtests.shared.KieServerDeployer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ApplyScorecardMultiModuleIntegrationTest extends PMMLApplyModelBaseTest {
+
+    private static Logger logger = LoggerFactory.getLogger(ApplyScorecardMultiModuleIntegrationTest.class);
+
+    private static ReleaseId releaseId = new ReleaseId("org.kie.server.testing", "scorecard_mm", "1.0.0.Final");
+
+    private static final String CONTAINER_ID = "scorecard_mm";
+
+    private static final long EXTENDED_TIMEOUT = 300000L;
```

Comment:
Are you sure about this value? 5 minutes of timeout?

Reply:
That is the value that was set by Jiri Petrlik in the ApplyScorecardModelIntegrationTest. I copied it from there.